packages/angular_bloc/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Angular Components that make it easy to implement the BLoC (Busines
 version: 3.0.0
 repository: https://github.com/felangel/bloc
 issue_tracker: https://github.com/felangel/bloc/issues
-homepage: https://bloclibrary.dev
+homepage: https://github.com/felangel/bloc/tree/master/packages/angular_bloc
```

Comment:
can you instead please update the `repository` rather than the `homepage`? Looks like `repository` takes precedence (https://github.com/dart-lang/pub-dev/issues/3288#issuecomment-581306275)